packages/dds/tree/src/feature-libraries/editable-tree/editableTreeContext.ts

```diff
@@ -154,10 +173,25 @@ export class ProxyContext implements EditableTreeContext {
         return this.getRoot(true);
     }
 
+    public set unwrappedRoot(value: ContextuallyTypedNodeData | undefined) {
+        const rootField = this.getRoot(false);
+        const mapTrees = applyFieldTypesFromContext(this.schema, rootField.fieldSchema, value);
+        const cursors = mapTrees.map(singleMapTreeCursor);
+        if (rootField.length > 0) {
+            rootField.replaceNodes(0, cursors);
```

Comment:
I'm guessing (let me know if that's not the case) that this setter is meant to have the same merge semantics as `replaceField` does.
If that's the case, it would be good to document the fact that we're using `replaceNodes` despite it not being what we really want.
Something like this:

`replaceNodes` has different merge semantics than replacing the root field would ideally offer:
`replaceNodes` should not overwrite concurrently inserted content while replacing the root field should.
We currently use `replaceNodes` here because the low-level editing API for the desired field replacement semantics are not yet avaialble.
TODO: update implementation once the low-level editing API is available.

Reply:
That's actually reveals a potential difference between `unwrappedRoot` and `root`. Whereas setting the `root` definitely means `replaceField`, setting `unwrappedRoot` might probably mean `replace its nodes` in cases, when the root field is not unwrapped to itself (only "a sequence root" case). I'll document all that.